python/lsst/meas/astrom/sip/__init__.py

```diff
@@ -21,11 +21,7 @@
 #
 
 import lsst.afw.geom  # for Box2I
-
-from .leastSqFitter1d import *
-from .leastSqFitter2d import *
-from .createWcsWithSip import *
-from .matchSrcToCatalogue import *
+from .._meas_astromLib import *
```

Comment:
I don't think the `sip` namespace should lift all of the library functions, but just the sip ones.